src/main/java/utils/Storage.java

```diff
@@ -0,0 +1,77 @@
+package utils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+
+public class Storage {
+
+    private String filePath;
+    private File file;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        this.file = new File(filePath);
+    }
+
+    /**
+     * Returns ArrayList of Tasks loaded from filePath location
+     */
+    public ArrayList<Task> load() {        
+        ArrayList<Task> tasks = new ArrayList<>();
+        
+        try {
+            Scanner dataReader = new Scanner(this.file);
+
+            while (dataReader.hasNextLine()) {
+                String[] data = dataReader.nextLine().split("\\|");
+                String type = data[0];
+                boolean isDone = data[1] == "1" ? true : false;
```

Comment:
Need not explictly return true/false
```suggestion
                boolean isDone = data[1] == "1";
```

Reply:
Nice catch! Thanks